### PR TITLE
[FIRRTL][LowerClasses] Handle InstanceTarget.

### PR DESCRIPTION
**Pull request by dtzSiFive**
Reject if does not target what it should.

### CODE REVIEW

File: lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp [hunk 1]
```diff
@@ -791,6 +791,8 @@ struct PathOpConversion : public OpConversionPattern<firrtl::PathOp> {
     if (!pathInfo) {
       if (op.getTargetKind() == firrtl::TargetKind::DontTouch)
         return emitError(op.getLoc(), "DontTouch target was deleted");
+      if (op.getTargetKind() == firrtl::TargetKind::Instance)
+        return emitError(op.getLoc(), "Instance target was deleted");
```
Comment (youngar) [inline lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp:795]: Should we add a test for this?

**Review by youngar — APPROVED** [review]
LGTM thanks!

**Review by mikeurbach — APPROVED** [review]
Makes sense to me.

File: lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp (hunk 1 above)
Comment (dtzSiFive) [reply lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp:795]: Good call, added, thanks!

**Pull request merged**